oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticConnectionRule.java

```diff
@@ -130,6 +139,46 @@ private void downloadSimilaritySearchPluginIfNotExists(String localPluginPath, S
         }
     }
 
+
+
+    private ElasticConnectionConnectionModel initializeElasticConnectionModel (String elasticConnectionString) {
```

Comment:
this method and the overloaded version below initialize an instance variable. The return values are never used. Should we remove them?
```suggestion
    private void initializeElasticConnectionModel (String elasticConnectionString) {
```

Reply:
removed returning model.